articles/azure/azs-how-create-service-fabric-portal.md

```diff
@@ -158,11 +160,23 @@ You also need the thumbprint of the certificate(s). For information about how to
 
    - **Repair Manager** - Select **Yes** to use Repair Manager to enable patch orchestration on node types with a durability of bronze, which helps keep your VMs up to date.
 
-   - **Service Fabric deployment package URL** - Specify the URL to download the Service Fabric deployment package from. For off-line scenarios, download the service fabric package from the URL specified and upload it to a blob and enable anonymous access and specify the URL here.
+   - **Diagnostic Data Age In Days** - Specify the days to keep the Service Fabric diagnostic log in the diagnostics store.
+
+   - **Service Fabric deployment package URL** - Specify the URL to download the Service Fabric deployment package from. For disconnected scenarios, download the service fabric package from the URL specified and upload it to a blob and enable anonymous access and specify the URL here.
+
+   - **Service Fabric runtime package URL** - Specify the URL to download the Service Fabric runtime package from. For disconnected scenarios, download the service fabric runtime from the URL specified and upload it to a blob and enable anonymous access and specify the URL here.
+
+   > [!WARNING]
+    > There are compatibility issues between the latest release of Service Fabric and its corresponding SDK. Until that issue is addressed, please provide the following parameters to the deployment package URL and runtime package URL. Your deployments will fail otherwise.
+    >
+    > - Service Fabric deployment package URL: https://download.microsoft.com/download/8/3/6/836E3E99-A300-4714-8278-96BC3E8B5528/6.5.641.9590/Microsoft.Azure.ServiceFabric.WindowsServer.6.5.641.9590.zip
+    > - Service Fabric runtime package URL: https://download.microsoft.com/download/B/0/B/B0BCCAC5-65AA-4BE3-AB13-D5FF5890F4B5/6.5.641.9590/MicrosoftAzureServiceFabric.6.5.641.9590.cab
+    >
+    > For disconnected deployments, download these packages from the specified location and host it locally on an Azure Stack Hub Blob.

```

Comment:
packages is plural so it should probably be ...host them locally...

Reply:
@cblackuk Is the "it" referring to the packages or the service fabric? If packages, then it should be "them".